app/jobs/old_records_cleanup_job.rb

```diff
@@ -0,0 +1,25 @@
+class OldRecordsCleanupJob < ApplicationJob
```

Comment:
Drop this class for now. No need to purge data after retention period. We're just building the foundation.